src/sbt-test/sbt-scalafix/cross-build/test

```diff
@@ -4,18 +4,18 @@
 -> scalafixAll --test ProcedureSyntax
 -> scalafix --test ProcedureSyntax
 -> compile:scalafix --test ProcedureSyntax
--> test:scalafix --test ProcedureSyntax
+-> Test/scalafix --test ProcedureSyntax
```

Comment:
As is, this PR seems to only handle shell commands on the Test config, so we would continue to have warnings (and eventually errors). Maybe the regexp you use could be updated?